src/editor/ui_menus/player_menu.cc

```diff
@@ -135,21 +136,8 @@ EditorPlayerMenu::EditorPlayerMenu(EditorInteractive& parent,
 	box_.add(&no_of_players_, UI::Box::Resizing::kFullSize);
 	box_.add_space(2 * kMargin);
 
+	assert(!parent.finalized());
 	const Widelands::Map& map = eia().egbase().map();
-
-	// Ensure that there is at least 1 player
```

Comment:
This code was placed here to fix a bug (I don't remember which one). Don't remove it without a good reason.

Reply:
This was merely shifted to the EIA which now creates an undeletable player directly on initialization

Reply:
All good then. Maybe add an assert here to ensure that whatever the bug was won't regress?